modules/catalog/src/test/java/org/apache/ignite/internal/catalog/CatalogManagerRecoveryTest.java

```diff
@@ -115,9 +114,10 @@ private void createComponents() {
     }
 
     private void startComponentsAndDeployWatches() {
-        Stream.of(metaStorageManager, catalogManager).forEach(IgniteComponent::start);
-
-        assertThat(metaStorageManager.deployWatches(), willCompleteSuccessfully());
+        assertThat(
+                allOf(metaStorageManager.start(), catalogManager.start(), metaStorageManager.deployWatches()),
```

Comment:
Should we start the components (and wait for futures to complete) and only then deploy watches? Or will `deployWatches()` care about this itself?

Reply:
Fix it